client/components/account-balances/balances-tab-panel.tsx

```diff
@@ -35,6 +36,7 @@ type BalanceTab = {
 	availableFunds: number;
 	pendingFunds: number;
 	delayDays: number;
+	instant?: AccountOverview.InstantBalance;
```

Comment:
Nit: I'd suggest naming this `instantBalance` or `instantFunds` for clarity. I thought `instant` represented a `boolean` at first glance.

Reply:
Good point. Addressed in [7115c26](https://github.com/Automattic/woocommerce-payments/pull/6004/commits/7115c26263b2c819af8cdd4f8cdb3110b830ac3e).